app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name" translate="false">AntennaPod</string>
+    <string name="pref_nextcloud_gpodder_connect_title">connect</string>
```

Comment:
Please add strings to the core module where the other strings are defined. That's the file that gets translated by volunteers. Also, please start strings with a capital letter and (if possible) try to re-use existing strings